lib/crypto/Crypto.js

```diff
@@ -72,7 +72,19 @@ module.exports = class Crypto {
 		const power = new BigNumber(key.d());
 		const mod = new BigNumber(key.n());
 
-		return (modPow(base, power, mod)).toBEBuffer().toString('base64');
+		//Somehow sometimes we have a 514 byte hex key that starts with "00". In that case use only the last 512 bytes
+		var hexString = modPow(base, power, mod).toBEBuffer().toString('hex');
+		if(hexString.substr(0,2) === "00" && hexString.length == 514) 
+		{
+			console.log("hex string of key starts with \"00\" and is 514 bytes long, fixing it to be 512 bytes long by stripping leading \"00\"");
+			hexString = hexString.substr(2);
+			var base64String = Buffer.from(hexString, 'hex').toString('base64')
+			return base64String;
+		}
+		else
```

Comment:
can we remove the else statement, seems unnecessary here.

Reply:
I guess I could tidy it up a bit so we use hexString in every case, so we don't need the else if that's what you're after?